scripts/jobs/housing/housing_mtfh_case_notes_enriched.py

```diff
@@ -111,7 +111,7 @@ def main():
     mmh_notes_df = mmh_notes_df.drop(columns={'all_notes_no_punct', 'all_notes_no_punct_no_stop'})
     mmh_notes_df = add_import_time_columns_pandas(mmh_notes_df)
     mmh_notes_df.to_parquet(path=s3_output_location, partition_cols=PARTITION_KEYS)
-    glue_client.start_crawler(Name='Housing MTFH case notes enriched to refined')
+    glue_client.start_crawler(Name='housing-mtfh-case-notes-enriched-to-refined')
     logger.info(f'Refined casenotes written to {s3_output_location}')
```

Comment:
Thanks, Anna. I've double-checked the name in the log; it's your revised "`housing-mtfh-case-notes-enriched-to-refined`".

I've rechecked the source code of the module again - the crawler "name" is not an input name, so entering "`name = "${local.short_identifier_prefix}Housing MTFH case notes enrichment to refined`"" will be ignored by Terraform. Terraform doesn't throw an error for undeclared variables by default; it ignores them. Thus, the `name` specified earlier version is not working.

By default, it will use job_name_identifier = replace(lower(var.job_name), "/[^a-zA-Z0-9]+/", "-") as the crawler name, not allowing us to change it based on the documentation at https://github.com/LBHackney-IT/Data-Platform/blob/main/terraform/modules/aws-glue-job/03-inputs-derived.tf.

Now, all is good. Thanks for fixing it.

Reply:
the module outputs the crawler name, so you could have used that and imported it as an environment variable. 

don't think it's worth changing, but maybe helpful in the future :)

Reply:
Well spotted.